src/main/resources/org/cbioportal/persistence/mybatisclickhouse/StudyViewFilterMapper.xml

```diff
@@ -13,6 +13,26 @@
                     #{studyId}
                 </foreach>
             </if>
+
+
+            <if test="studyViewFilter.genomicProfiles != null and !studyViewFilter.genomicProfiles.isEmpty()">
```

Comment:
@gblaih here is the filter.  i found a small problem with foreach at the bottom but this is the basic idea.